### DISCUSSION TITLE
[FE] axios

### DISCUSSION THREAD

**Discussion by Rinasham** [Hackathon5-TeamD/evolution-quest#37] [Q&A]
どこかのタイミングでpackage.jsonにaxiosもよろしくお願いしたい所存

**Comment by Terralien-jp** [comment]
あー要りますね！
前回使い方わからずに自爆したので教えてもらいたい所ｗ

**Reply by Rinasham** [reply]
モックの使い方についてはいまいちよく分かってません😇

いつも代わりにJsonPlaceholder使ってるのでみんなでモックで自爆したらそっち使うのもありかもしれませぬ

**Reply by haru-24** [reply]
@Terralien-jp  @Rinasham 
もしモックを使うのでしたらStudyCalendarのじゃなくて`JSON Server`というライブラリの方が簡単そうでした🤔
https://www.webprofessional.jp/mock-rest-apis-using-json-server/

**Reply by Terralien-jp** [reply]
拝見します！（本当に拝見するだけ）

**Reply by Terralien-jp** [reply]
テスト段階で試してみましょうｗ

**Comment by Terralien-jp** [comment]
コレ多分issue化案件です！

**Comment by kobakichi** [comment]
インストールする前にissue立てます！！
